app/src/main/res/values/attrs.xml

```diff
@@ -23,7 +23,8 @@
         <attr name="inputDefaultStrokeWidth" format="dimension" />
         <attr name="inputDefaultStrokeColor" format="reference|color" />
         <attr name="popupMenuBackground" format="reference|color" />
-        <attr name="accentShape" format="reference" />
+        <attr name="accentCircle" format="reference" />
+        <attr name="backgroundCircle" format="reference" />
```

Comment:
Those attributes are not changed by any theme (except fort the default) so they can be omitted.